.swiftlint.yml

```diff
@@ -42,69 +30,16 @@ whitelist_rules:
   # Prefer `() -> ` over `Void -> `.
   - empty_parameters
 
-  # Files should not span too many lines.
-  # - file_length
-
-  # Force casts should be avoided.
-  # - force_cast
-
-  # Force tries should be avoided.
-  # - force_try
-
-  # Force unwrapping should be avoided.
-  # - force_unwrapping
```

Comment:
Do we get a lot of violations from those rules? I feel they're quite important. Perhaps good to create an issue at least to track it, as I'd hope we can fix them asap.